### PR TITLE
Fix misleading error message on file import

### PR DESCRIPTION
**Pull request by gregorjerse**
if status code indicating error was received from the upstream server

### CODE REVIEW

File: resolwe/process/fields.py [hunk 1]
```diff
@@ -624,13 +624,14 @@ def importUncompressed():
...
 
-        # Large file download from Google Drive requires cookie and token.
         try:
             response = None
+            # Large file download from Google Drive requires cookie and token.
             if re.match(
                 r"^https://drive.google.com/[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]$",
                 src,
             ):
+                print("Importing file from google drive.")
```
Comment (dblenkus) [inline resolwe/process/fields.py:634]: Remove print.

File: resolwe/process/fields.py [hunk 2]
```diff
@@ -648,12 +649,14 @@ def importUncompressed():
                 r"^(https?|ftp)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]$",
                 src,
             ):
+                print("Importing file from http/ftp server")
```
Comment (dblenkus) [inline resolwe/process/fields.py:652]: Remove print.

File: resolwe/process/fields.py (hunk 1 above)
Comment (gregorjerse) [reply resolwe/process/fields.py]: Done.

File: resolwe/process/fields.py (hunk 2 above)
Comment (gregorjerse) [reply resolwe/process/fields.py]: Done.

**Review by dblenkus — APPROVED** [review]

**Pull request merged**